app/web/src/store/module.store.ts

```diff
@@ -439,8 +441,7 @@ export const useModuleStore = () => {
 
             return new ApiRequest<{ id: string }>({
               method: "post",
-              url: "/module/export_workspace",
-              params: { ...getVisibilityParams() },
+              url: WORKSPACE_API_PREFIX.concat(["export"]),
```

Comment:
Getting this error in the frontend:

Uncaught (in promise) Error: Bad URLPattern v2,workspaces,[object Object],change-sets,[object Object],modules with: [object Object]
    at pinia_api_tools.ts:184:27
    at Array.forEach (<anonymous>)
    at describePattern (pinia_api_tools.ts:178:11)
    at triggerApiRequest (pinia_api_tools.ts:356:27)
    at fireActionResult (pinia_api_tools.ts:541:35)
    at Proxy.wrappedActionFn (pinia_api_tools.ts:568:17)
    at async Proxy.callback (module.store.ts:477:17)
(anonymous) @ pinia_api_tools.ts:184
describePattern @ pinia_api_tools.ts:178
triggerApiRequest @ pinia_api_tools.ts:356
fireActionResult @ pinia_api_tools.ts:541
wrappedActionFn @ pinia_api_tools.ts:568
await in wrappedActionFn
callback @ module.store.ts:477
(anonymous) @ realtime.store.ts:280
(anonymous) @ chunk-PNCXOGCT.js?v=cd3386e6:186
baseForOwn @ chunk-PNCXOGCT.js?v=cd3386e6:401
(anonymous) @ chunk-PNCXOGCT.js?v=cd3386e6:412
forEach @ lodash-es.js?v=dca336af:2640
handleEvent @ realtime.store.ts:278
(anonymous) @ realtime.store.ts:294
ReconnectingWebSocket2._callEventListener @ reconnecting-websocket.js?v=90d67f5e:521
(anonymous) @ reconnecting-websocket.js?v=90d67f5e:162
ReconnectingWebSocket2._handleMessage @ reconnecting-websocket.js?v=90d67f5e:161
UserInteractionInstrumentation2._invokeListener @ @opentelemetry_instrumentation-user-interaction.js?v=7c015b2c:187
patchedListener @ @opentelemetry_instrumentation-user-interaction.js?v=7c015b2c:226Understand this errorAI

Reply:
interesting! Will look now and see